src/domain/wallets/tx-history.ts

```diff
@@ -117,12 +116,12 @@ export const fromLedger = (
             ...baseTransaction,
             initiationVia: {
               type: PaymentInitiationMethod.IntraLedger,
-              counterPartyWalletPublicId: walletPublicId as WalletPublicId,
+              counterPartyWalletId: walletId as WalletId,
```

Comment:
This one's wrong, because we originally named `walletPublicId` badly in the `LedgerTransaction` type (see related comment). It should actually be the `recipientWalletId` property.